src/fn/edit-contacts.js

```diff
@@ -7,18 +7,34 @@ const pouch = require('../lib/db');
 const safeStringify = require('../lib/safe-stringify');
 const toDocs = require('./csv-to-docs');
 const EDIT_RESERVED_COL_NAMES = [ 'parent', '_id', 'name', 'reported_date' ];
-const DOC_TYPES = ['district_hospital', 'health_center', 'clinic', 'person', 'user', 'user-settings', 'contact'];
 const DOCUMENT_ID =  'documentID';
+const userPrompt = require('../lib/user-prompt');
+const fetchDocumentList = require('../lib/fetch-document-list');
+const OVERWRITE_ALL_FILES_OPTION = 1;
+const CANCEL_OVERWRITE_OPTION = 2;
+
+const jsonDocPath = (directoryPath, docID) => `${directoryPath}/${docID}.doc.json`;
+
+// check to see if we should write/overwrite the file
+const overwriteFileCheck = (doc, args, overwriteAllFiles) => {
+  return args.updateOfflineDocs || args.force || overwriteAllFiles || !fs.exists(jsonDocPath(args.docDirectoryPath, doc._id));
+};
+
+const saveJsonDoc = (doc, args) => {
+  const writeFile = (writeDoc) => fs.write(jsonDocPath(args.docDirectoryPath, writeDoc._id), safeStringify(writeDoc) + '\n');
+  
+  return writeFile(doc);
```

Comment:
It seems strange to me that we create this `writeFile` function and then never use it, except to call it with the parameters we received. 
```suggestion
return fs.write(jsonDocPath(args.docDirectoryPath, doc._id), safeStringify(doc) + '\n');
```